packages/framework/src/Model/Sitemap/SitemapCronModule.php

```diff
@@ -1,14 +1,14 @@
 <?php
 
-namespace Shopsys\FrameworkBundle\Component\Sitemap;
+namespace Shopsys\FrameworkBundle\Model\Sitemap;
```

Comment:
@PetrHeinz had some good points in the analysis - it does not belong to `Model` strictly...but for me it is good enough now. However, you could probably create a user story for extracting the sitemap into components again (the similar way the friendly urls generating was rewritten)